### PR TITLE
Add team description size message

### PR DESCRIPTION
**Pull request by Sanket2410**
Add character limit for description property of team

### CODE REVIEW

File: api-reference/beta/resources/team.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Every team is associated with a [group](../resources/group.md). The group has th
 | Property | Type | Description |
 |:---------------|:--------|:----------|
 |displayName|string| The name of the team. |
-|description|string| An optional description for the team. |
+|description|string| An optional description for the team. Max. length: 1024 characters. |
```
Comment (AkJo) [inline api-reference/beta/resources/team.md:52]: Maximum?

Comment (AkJo) [reply api-reference/beta/resources/team.md:52]: @Lauragra , what is the guidance for specifying constraints

Comment (Lauragra) [reply api-reference/beta/resources/team.md:52]: Yes, please spell out as maximum.

**Review by AkJo — APPROVED** [review]

File: api-reference/beta/resources/team.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ Every team is associated with a [group](../resources/group.md). The group has th
 | Property | Type | Description |
 |:---------------|:--------|:----------|
 |displayName|string| The name of the team. |
-|description|string| An optional description for the team. |
+|description|string| An optional description for the team. Maximum length: 1024 characters. |
```
Comment (AkJo) [inline api-reference/beta/resources/team.md:52]: Please make the same update to v1 documentation as well.

**Comment by Sanket2410** [comment]
Hey @Lauragra , can you please merge this PR.

**Pull request merged**